src/main/java/bio/terra/cli/command/notebook/Create.java

```diff
@@ -0,0 +1,110 @@
+package bio.terra.cli.command.notebook;
+
+import bio.terra.cli.app.AuthenticationManager;
+import bio.terra.cli.app.ToolsManager;
+import bio.terra.cli.model.GlobalContext;
+import bio.terra.cli.model.TerraUser;
+import bio.terra.cli.model.WorkspaceContext;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra notebook create" command. */
+@CommandLine.Command(name = "create", description = "Create a new AI Notebook instance within your workspace.")
+public class Create implements Callable<Integer> {
+
+  @CommandLine.Parameters(
+      index = "0",
+      description =
+          "The unique name to give to the notebook instance. Cannot be changed later. "
+              + "The instance name must be 1 to 63 characters long and contain only lowercase "
+              + "letters, numeric characters, and dashes. The first character must be a lowercase "
+          + "letter and the last character cannot be a dash.")
+  private String instanceName;
+
+  @CommandLine.Option(
+      names = "location",
+      defaultValue = "us-central1-a",
+      description = "The Google Cloud location to create the instance within, e.g. us-central1-a")
+  private String location;
+
+  @CommandLine.Option(
+      names = "machine-type",
+      defaultValue = "n1-standard-4",
+      description = "The Compute Engine machine type of this instance.")
+  private String machineType;
+
+  @CommandLine.Option(
+      names = "vm-image-project",
+      defaultValue = "deeplearning-platform-release",
+      description = "The ID of the Google Cloud project that this VM image belongs to.")
+  private String vmImageProject;
```

Comment:
How is this different than the workspace project id -- is it just where the GCR repository lives?

Reply:
The project where the GCE image lives. I think but am not sure if it's in GCR. The defaults are the AI Platform folks default images. https://cloud.google.com/ai-platform/notebooks/docs/reference/rest/v1/VmImage